src/routing.tsx

```diff
@@ -18,6 +17,7 @@ import Sales from 'views/Statistics/Sales';
 import Taxes from 'views/Statistics/Taxes';
 import Stocks from 'views/Warehouse/Stocks';
 import Supplies from 'views/Warehouse/Supplies';
+import Categories from 'views/Menu/Categories';
```

Comment:
The imports were sorted alphabetically in this file, this is just another argument to automatically sort the imports with the plugin.

Reply:
We can make a ticket which you pointed above to extend Prettier in the future.

Reply:
It is still not sorted properly ;) Nevermind, we will fix that with plugin later.